src/common/filehistorycmn.cpp

```diff
@@ -135,20 +136,43 @@ void wxFileHistoryBase::AddFileToHistory(const wxString& file)
     m_fileHistory.insert(m_fileHistory.begin(), file);
     numFiles++;
 
-    // update the labels in all menus
+    RefreshLabels();
+}
+
+void wxFileHistoryBase::RefreshLabels()
+{
+    size_t i;
+    size_t numFiles = m_fileHistory.size();
```

Comment:
Another pretty minor improvement
```suggestion
    const size_t numFiles = m_fileHistory.size();
```